_test_unstructured_client/test__decorators.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
+from unstructured_client import UnstructuredClient
+from unstructured_client.models import shared
+from unstructured_client.models.errors import SDKError
+
+
+FAKE_KEY = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+
+
+@pytest.mark.parametrize(
+    "server_url",
+    [
+        # -- well-formed url --
+        "https://unstructured-000mock.api.unstructuredapp.io",
+        # -- common malformed urls --
+        "unstructured-000mock.api.unstructuredapp.io",
+        "http://unstructured-000mock.api.unstructuredapp.io/general/v0/general",
+        "https://unstructured-000mock.api.unstructuredapp.io/general/v0/general",
+        "unstructured-000mock.api.unstructuredapp.io/general/v0/general",
+    ],
+)
+def test_clean_server_url_fixes_malformed_paid_api_url(server_url: str):
+    client = UnstructuredClient(
+        server_url=server_url,
+        api_key_auth=FAKE_KEY,
+    )
```

Comment:
These will all fit on one line, won't they? Vertical space is something to preserve on principle. Leaving off the trailing comma on the last argument will keep black from automatically formatting it this way. Each of these tests uses up more space than necessary.

Reply:
That was made from using black, but I can revert it